contracts/auctions/src/contract.rs

```diff
@@ -0,0 +1,430 @@
+use soroban_sdk::{contract, contractimpl, log, vec, Address, Env, Vec};
+
+use crate::{
+    collection,
+    error::ContractError,
+    storage::{
+        generate_auction_id, get_admin, get_auction_by_id, get_auctions, get_auctions_by_seller_id,
+        get_highest_bid, is_initialized, save_admin, save_auction_by_id, save_auction_by_seller,
+        set_highest_bid, set_initialized, update_admin, update_auction, validate_input_params,
+        Auction, AuctionStatus, ItemInfo,
+    },
+    token,
+};
+const ONE: u64 = 1u64;
```

Comment:
looks weird, I know, I don't want to hardcode 1 and forget about it - reason for that is that currently the marketplace allows to handle only auctions for a single item, not multiple items.

Let's say a seller has 5 items, on the marketplace they can sell just one 1, hence `const ONE`. 

I guess the next improvement can be allowing to sell entire collections, among the other type of auctions

Reply:
> Let's say a seller has 5 items, on the marketplace they can sell just one 1, hence const ONE.

I still don't get what's better in getting a constant instead of a number that literally represent that word?

Reply:
removed